<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxLAB">
     <property name="text">
      <string>对标签使用换行</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxINT">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>标签之间直接换行（代码美观，但可能导致外观改变）</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxIND">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>使用缩进</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxLAB</tabstop>
  <tabstop>checkBoxINT</tabstop>
  <tabstop>checkBoxIND</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLAB</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxINT</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLAB</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxIND</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
